apps/condo/domains/meter/hooks/useCreateMeterModal.tsx

```diff
@@ -17,8 +18,13 @@ export function useCreateMeterModal (organizationId: string, propertyId: string,
 
     const handleMeterCreate = useCallback(values => {
         const numberOfTariffs = values.numberOfTariffs || 1
+        const number = String(get(values, 'number', '')).trim()
+        const accountNumber = String(get(values, 'accountNumber', '')).trim()
```

Comment:
Why explicit cast to string?

Reply:
you're right, it's not needed here. Removed it